roles/artifacts/defaults/main.yml

```diff
@@ -18,4 +18,7 @@
 # All variables intended for modification should be placed in this file.
 # All variables within this role should have a prefix of "cifmw_artifacts"
 cifmw_artifacts_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_artifacts_crc_host: "api.crc.testing"
+cifmw_artifacts_crc_user: "core"
+cifmw_artifacts_crc_sshkey: "~/.crc/machines/crc/id_ecdsa"
```

Comment:
do we have this key on extracted layout? we may want to override it in such case to use zuul's key instead.

Reply:
it's in the job definition I think

Reply:
key exists, and is working, we're all good.